### PR TITLE
Support CPC extension

### PR DESCRIPTION
**Pull request by AshleyYangSZ**

### CODE REVIEW

File: src/popup/config/config.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "name": "Intune",
   "help": "https://intunewiki.com/wiki/Intune_Azure_Portal",
   "changelog": [
-	{
+    {
       "version": "4.6",
```
Comment (PxlBuzzard) [inline src/popup/config/config.json:12]: Make sure to bump to `"version": "4.7",` so that the extension knows to use the updated config.

Comment (PxlBuzzard) [reply src/popup/config/config.json:12]: Along with that, add a bit of text to the changelog to explain what `4.7` does.

File: src/popup/config/config.json [hunk 2]
```diff
@@ -338,6 +338,44 @@
               "sideloadUrl": "https://localhost:17159/intuneworkflows"
             }
           ]
+        },
+        {
+          "name": "Microsoft_Azure_CloudPC",
+          "environments": [
+            {
+              "label": "Int",
```
Comment (PxlBuzzard) [inline src/popup/config/config.json:361]: Not a blocking comment: While it's fine to use any name that you'd like, naming it `Dogfood` will keep it grouped with all of the other `Dogfood` extensions.

**Review by PxlBuzzard — CHANGES_REQUESTED** [review]
Looks good! Just make sure to bump the version and it'll be good to merge :)

**Comment by AshleyYangSZ** [comment]
Thanks Daniel,
Sorry for late response. I followed the readme and try to test locally, but the added Microsoft_Azure_CloudPC extension and int env didn't show. Do you have any idea?
***@***.***

Thanks,
Ashley

File: src/popup/config/config.json [hunk 3]
```diff
@@ -77,9 +82,19 @@
...
+            "feature.canmodifystamps": "true",
+            "feature.canmodifyextensions": "true",
+            "feature.intune_env": "selfhost",
+            "feature.CloudPCGraphVersion": "testprodbeta_cpc_int"
+          }
+        },
         {
           "label": "Selfhost",
-          "host": "endpoint.microsoft.com",
+          "host": "intune.microsoft.com",
```
Comment (AshleyYangSZ) [inline src/popup/config/config.json:97]: @PxlBuzzard Currently Intune portal changed to intune.microsoft.com so update this as well

File: src/popup/config/config.json (hunk 2 above)
Comment (AshleyYangSZ) [reply src/popup/config/config.json:361]: As CloudPC is a new extension and INT is our internal dev testing environment, integrated with Intune externsion's SH environment. It's not dogfood extension, so I prefer to INT.

File: src/popup/config/config.json (hunk 1 above)
Comment (AshleyYangSZ) [reply src/popup/config/config.json:12]: Really a good comment. Updated.

**Review by PxlBuzzard — APPROVED** [review]

**Pull request merged**